lib/main_common.pm

```diff
@@ -1710,6 +1710,7 @@ sub load_extra_tests_console {
     loadtest 'console/libgcrypt' if ((is_sle(">=12-SP4") && (check_var_array('ADDONS', 'sdk') || check_var_array('SCC_ADDONS', 'sdk'))) || is_opensuse);
     loadtest "console/gd";
     loadtest 'console/valgrind' unless is_sle('<=12-SP3');
+    loadtest 'console/wpa_supplicant' if (is_x86_64 && (is_sle('15+') || is_leap('15.1+') || is_tumbleweed));
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

Reply:
I'll change this issue in the afternoon, thanks for reminding me about the logic.

I understand your point about the test suites but don't have a sufficient overview yet. Do you have a recommendation where to put the test? Would be happy to apply the changes then.

Reply:
Changed it to `loadtest 'console/wpa_supplicant' unless (!is_x86_64 || is_sle('<15') || is_leap('<15.1'));`